index.html

```diff
@@ -242,7 +242,45 @@ <h1 class="headline-h1">About Me</h1>
         </ul>
       </li>
     </ul>
-  </div>                              
+  </div>
+  <div class="ContactPage">
+   <form action="https://formspree.io/f" method="post"></form>
```

Comment:
Please ensure that all input fields are wrapped in a single `form` element according to the requirement. And please ensure that form submission is working as expected.
Example:
`<form>`
       `<input>......</input>`
`</form>`